docs/detections/rules-ui-create.asciidoc

```diff
@@ -215,6 +215,11 @@ image::images/eql-rule-query-example.png[]
 +
 NOTE: For sequence events, the {security-app} generates a single alert when all events listed in the sequence are detected. To see the matched sequence events in more detail, you can view the alert in the Timeline, and, if all events came from the same process, open the alert in Analyze Event view.
 +
+. (Optional) Click the EQL settings icon (image:images/eql-settings-icon.png[EQL settings icon,16,16]) to configure additional fields used by {ref}/eql.html#specify-a-timestamp-or-event-category-field[EQL search]:
+  * *Event category field*: Contains the event classification, such as `process`, `file`, or `network`. This field is typically mapped as a field type in the keyword family. Defaults to the `event.category` ECS field.
```

Comment:
Could be my own ignorance, but I'm not sure what is meant by "This field is typically mapped as a field type in the keyword family", I guess it means that there are some field types that are of the `keyword` type, and this field is usually mapped as one of those, but I'm not sure where I would go to understand that better. If you think it's clear, feel free to ignore this.

Reply:
@benironside I agree this is getting into some nitty-gritty Elasticsearch details, but I think readers who want to have this level of control over their data (customizing fields instead of just using ECS standards) would probably be familiar with field types and the keyword family, which AFAIK is exactly as you guessed. 

I considered adding a link to the Elasticsearch's [keyword family](https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html) docs, but again it seems a little unnecessary for the intended audience.

Reply:
Sounds good, I'll add it in the next commit...